components/Snackbar/src/private/MDCSnackbarMessageViewInternal.h

```diff
@@ -51,14 +51,26 @@ typedef void (^MDCSnackbarMessageDismissHandler)(BOOL userInitiated,
 @property(nonatomic, nullable, readonly, strong) MDCSnackbarMessage *message;
 
 /**
- If the snackbar view should be anchored to the bottom of the screen. Default is YES.
+ If the Snackbar view should be anchored to the bottom of the screen. Default is YES.
  */
-@property(nonatomic) BOOL anchoredToScreenEdge;
+@property(nonatomic) BOOL anchoredToScreenBottom;
 
 /**
- Creates a snackbar view to display @c message.
+ The layout constraint which determines how far the Snackbar is from the leading edge of the screen.
+ It is active when the alignment of the parent overlay view is MDCSnackbarAlignmentLeading.
+ */
+@property(nonatomic, nullable) NSLayoutConstraint *leadingConstraint;
```

Comment:
Super-nit: Please mark this property and `centerConstraint` as explicitly `strong`.

Reply:
I moved the constraints like you suggested doing in https://github.com/material-components/material-components-ios/pull/4163#discussion_r189891840 and I'm noticing that a lot of the strong properties surrounding them in their new location aren't explicitly declared strong. I'm torn between following your advice and marking these two constraints explicitly as strong and following Jeff's advice by keeping consistent with what's there 😬

Reply:
I think since it's moving to a class with existing properties, follow the norms there. My previous recommendation was based on my (incorrect?) observation that there seemed to be no previous precedent.